WalletWasabi.Fluent/Views/Wallets/Home/Tiles/WalletBalanceTileView.axaml

```diff
@@ -13,12 +14,23 @@
   <c:TileControl Title="BALANCE" IsBottomContentVisible="{Binding HasBalance^}" SeparatorMargin="0 0 0 6">
     <Viewbox Margin="0 3" VerticalAlignment="Center" MaxHeight="40">
       <c:PrivacyContentControl Classes="h2 monoSpaced" HorizontalContentAlignment="Center"
-                               Content="{Binding BalanceBtc^}" />
+                               Content="{Binding BalanceBtc^, Converter={x:Static converters:MoneyConverters.ToBtc}}">
+        <Interaction.Behaviors>
+          <b:CopyContentToClipboardBehavior Content="{Binding BalanceBtc^}" FlyoutMessage="Copied!">
+            <b:CopyContentToClipboardBehavior.FlyoutMessageTemplate>
+              <DataTemplate x:DataType="x:String">
+                <TextBlock Text="{Binding}" TextAlignment="Center" />
+              </DataTemplate>
+            </b:CopyContentToClipboardBehavior.FlyoutMessageTemplate>
+          </b:CopyContentToClipboardBehavior>
+        </Interaction.Behaviors>
```

Comment:
It is just weird that you set the message in `FlyoutMessage` when you could simply set it in the TextBlock.